electron/renderer.js

```diff
@@ -76,13 +89,13 @@ class Renderer {
                     } else {
                         ipcRenderer.send('mocha-done', 'ping');
                     }
-                } catch(err) {
-                    console.log("FLOSS - caught inner exception:", err);
+                } catch(e) {
+                    console.log("[floss] caught inner exception:", e.message);
```

Comment:
one nice thing about this was having the stack trace in headful mode. not sure, but does just the message here maintain that?